### PR TITLE
Add step function

### PR DESCRIPTION
**Pull request by benalleng**
My attempt at rebasing the arrow and step functionality into our current dev HEAD

[Preview](https://saving-satoshi-git-fork-benalleng-add-step-3490f8-savingsatoshi.vercel.app/en/chapters/chapter-9/opcodes-5?dev=true)

### CODE REVIEW

**Comment by 0tuedon** [comment]
I noticed we can't step or Run when we get our answer and we get a Success message when I just keep changing the stack

**Comment by daviroo** [comment]
> I noticed we can't step or Run when we get our answer and we get a Success message when I just keep changing the stack

Let's go over it today on the call.  I think we need to align on what the best behaviour is.

**Comment by 0tuedon** [comment]
> > I noticed we can't step or Run when we get our answer and we get a Success message when I just keep changing the stack
> 
> Let's go over it today on the call. I think we need to align on what the best behaviour is.

Yh Looking forward to that

File: content/lessons/chapter-9/opcodes-5.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function OpCodes5({ lang }) {
         answerScript={['OP_ADD', 'OP_3', 'OP_EQUAL']}
         prePopulate
         showRunButtons
-        readOnly
+        //readOnly
```
Comment (0tuedon) [inline content/lessons/chapter-9/opcodes-5.tsx:27]: remove comment

**Review by 0tuedon — APPROVED** [review]
LGTM!

**Pull request merged**